models/articles.model.js

```diff
@@ -36,4 +37,20 @@ exports.fetchAllArticles = () => {
     return db.query(query).then(({rows}) => {
         return rows;
     });
+}
+
+exports.updateVotesByArticleId = (id, update) => {
+    console.log(update)
+        return db.query(`
+            UPDATE articles
+            SET votes = votes + $2
+            WHERE article_id = $1
+            RETURNING *;
+            `, [id, update.inc_votes])
+        .then(({ rows }) => {
+            if (rows.length === 0) return ""
+                
```

Comment:
If rows.length === 0 we would want to reject the promise here to be caught by an error handler.